airflow-core/src/airflow/cli/commands/dag_command.py

```diff
@@ -523,7 +523,7 @@ def dag_report(args) -> None:
         if bundle.name not in bundles_to_reserialize:
             continue
         bundle.initialize()
-        dagbag = DagBag(bundle.path, include_examples=False)
+        dagbag = DagBag(bundle.path, include_examples=False, bundle_version=bundle.version)
```

Comment:
It's possible that not all of the dags are parsed on the latest bundle version yet. This should probably be `None`, meaning the latest.

Reply:
Won't that mean that bundle.version will be none?

Reply:
Wait, I just realized this is reparsing everything vs reading from the db, so this should be fine as-is.